src/libs/actions/Policy.js

```diff
@@ -166,6 +166,37 @@ function isPolicyOwner(policy) {
     return _.keys(loginList).includes(policy.owner);
 }
 
+/**
+ * Is the user the member of the given policy?
+ *
+ * @param {Object} policy
+ * @returns {Boolean}
+ */
+function isPolicyMember(policy) {
+    return !isPolicyOwner(policy) && policy.role !== CONST.POLICY.ROLE.ADMIN;
+}
+
+/**
+ * @param {Object|null} linkedWorkspace - the workspace the report is on, null if the user isn't a member of the workspace
+ * @param {Object|null} report
+ * @returns {Boolean}
+ */
+function shouldDisableRename(linkedWorkspace, report) {
```

Comment:
This method location feels wrong to me. `Policy.shouldDisableRename` is as though we are deciding if the user is renaming a policy, but that's not the case there since we're dealing with renaming a report name.

I think this method should be in ReportUtils, and the first param should be report and the second param should be policy (not linkedWorkspace). Thoughts?

Reply:
@hungvu193 `we will need to use some function from Policy,` 
What is this?

The only function is `isPolicyMember`, which we can move to PolicyUtils.js and make it import Policy.js functions.

And in ReportUtils we can import PolicyUtils.js function of `isPolicyMember`

Reply:
Cool, also i think im okay with dependency cycle for lib files. Feels like its bound to happen there and I'm unsure if its not good to have dependency cycles in lib files 🤷‍♂️ 
Either way, this might be a discussion for open-source some other day.